setup.py

```diff
@@ -21,6 +21,7 @@
         "robotframework-seleniumlibrary",
         "robotframework-jupyterlibrary>=0.3.1",
         "robotframework-OpenShiftCLI==1.0.1",
+        "robotframework-openshift",
```

Comment:
I think we should specify version ==X.Y.Z until the library is stable

Reply:
okay, I'm setting it to 1.0.0